pages/linux/expr.md

```diff
@@ -0,0 +1,23 @@
+# expr
+
+> Evaluate expressions and manipulate strings.
+
+- Get string length:
+
+`expr length {{string}}`
+
+- Evaluate logical or math expression with an operator('+','-','*','&','|' and etc.), special symbols should be escaped:
+
+`expr {{ARG1}} {{operator}} {{ARG2}}`
```

Comment:
should ARG1 and ARG2 be numbers only? If so, the tokens should probably be renamed.